src/components/MapView/Layers/BoundaryDropdown.tsx

```diff
@@ -13,19 +13,30 @@ import {
   Theme,
```

Comment:
We have BoundaryDropdown and LayerDropdown both in the `Layers` folder which doesn't make too much sense. Think we can move it out to some place better?